packages/appstore/src/components/main-title-bar/asset-summary.tsx

```diff
@@ -27,7 +28,7 @@ const AssetSummary = observer(() => {
     const cr_account = !is_eu_user && !no_CR_account;
 
     //dont show loader if user has no respective regional account
-    if ((is_switching || is_logging_in) && (eu_account || cr_account)) {
+    if (((is_switching || is_logging_in) && (eu_account || cr_account)) || !is_landing_company_loaded) {
```

Comment:
```suggestion
    if (((is_switching || is_logging_in) && (eu_account || cr_account)) || !is_landing_company_loaded || !last_updated) {
```
for when the exchange-rate data is not available